mindbot.py

```diff
@@ -416,7 +416,49 @@ async def createcreaturecard(interaction: discord.Interaction, artwork : discord
 		print("Error on Create Crature Card:" + str(e))
 		await interaction.followup.send(f"I choked and had to abort.")
 
+@tree.command(name = "editcreature", description = "Edit a Creature Card.")
+async def editcreaturecard(interaction: discord.Interaction, name: str, cardset:str ):
+	try:
+		if (name):
+			userid = interaction.user.id
+
+			# The first answer must be given within 3sec.
+			await interaction.response.defer(ephemeral=True, thinking=True)
+			# From now on we have 15 minutes
+			
+			#Get all cards from user
+			db = mongodb["cardcreator"]
+			card = db["customcreatures"].find_one(filter={ "user_id": userid, "name": name ,"cardset": cardset})
+
+			if card is None:
+				await interaction.response.send_message(f'No Card found')
```

Comment:
you are missing a return if the card does not exist.

Reply:
now :D